monitoring/nemo_time_mean_temporalmap.py

```diff
@@ -60,8 +60,8 @@ def time_operation(self, varname, leg_cube):
         leg_average = iris.util.new_axis(leg_average, "time")
         leg_average = helpers.cubes.set_metadata(
             leg_average,
-            title=f"{leg_average.long_name.title()} (Annual Mean Map)",
-            comment=f"Leg Mean of **{varname}**.",
+            title=f"{leg_average.long_name} (annual mean map)",
+            comment=f"Leg mean of **{varname}**.",
```

Comment:
@uwefladrich should I keep the reference to the leg here or switch to "annual" in the comment as well?

Reply:
we have a similar case in [nemo_global_mean_year_mean_timeseries.py, l. 43](https://github.com/uwefladrich/scriptengine-tasks-ecearth/pull/78/files#diff-5d9171849be3d7bd0312b89120267d036838ec17f21547d23c2b97a1c7ea2621L43) (leg) vs. [oifs_global_mean_year_mean_timeseries.py, l. 124](https://github.com/uwefladrich/scriptengine-tasks-ecearth/pull/78/files#diff-d85267aa34f988f029a39ac17187065b377f6d244a8605275d3e36c69cd692dfR124) (year)

Reply:
Switch to "annual", I would say.